src/components/partials/socialLinks/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { DEV_PATH } from '@/lib/constants'
+
+export const SocialLinks = ({ node }): JSX.Element => {
+  const hostUrl = DEV_PATH + node?.path?.alias
+
+  return (
+    <div className="vads-u-margin-bottom--0p5 medium-screen:vads-u-margin-bottom--2">
+      <ul className="usa-unstyled-list vads-u-display--flex" role="list">
+        <li className="vads-u-margin-right--5 medium-screen:vads-u-margin-right--2p5">
+          <a
+            className="va-js-share-link"
+            href={`https://www.facebook.com/sharer/sharer.php?href=${hostUrl}`}
+          >
+            <i className="va-c-social-icon fab fa-facebook vads-u-margin-right--0p5"></i>
+            Share on Facebook
+          </a>
+        </li>
+
+        <li className="vads-u-flex--1">
+          <a
+            className="va-js-share-link"
+            href={`https://twitter.com/intent/tweet?text=${node?.title}&url=${hostUrl}`}
+          >
+            <i className="va-c-social-icon fab fa-twitter vads-u-margin-right--0p5"></i>
+            Share on Twitter
+          </a>
+        </li>
+      </ul>
+    </div>
```

Comment:
It looks good to me! Based on the code it looks like you built the facilities variant. I noticed there is a similar but slightly different version in the includes directory with some additional calendar logic. I'm not sure where it's being used but I think we might want to make a note of it and consolidate them into a single component in the future. nice work! 🚀 

```
<a
data-description="{{ fieldDescription }}"
data-end="{{ fieldDatetimeRangeTimezone.endValue }}"
data-location="{{ fieldAddress.addressLine1 }} {{ fieldAddress.locality }}, {{ fieldAddress.administrativeArea }}"
data-subject="{{ title }}"
href="{{ entityUrl.path }}"
id="add-to-calendar-link" data-start="{{fieldDatetimeRangeTimezone.value }}"
>
<i class="va-c-social-icon fas fa-calendar-check vads-u-margin-right--0p5" aria-hidden="true" role="presentation" aria-hidden="true"></i>
Add to Calendar
</a>
```

1. [/facilities/story_social_share.drupal.liquid ](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/facilities/story_social_share.drupal.liquid )
2. [/includes/social-share.drupal.liquid](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/includes/social-share.drupal.liquid)

Reply:
Yes, and also here - campaign_landing_page.drupal.liquid
Per the story I based the component off of the facilities variant and it looks like it will be able to use in the other two areas.